tests/components/devolo_home_network/test_config_flow.py

```diff
@@ -77,14 +77,30 @@ async def test_form_error(hass: HomeAssistant, exception_type, expected_error) -
     ):
         result2 = await hass.config_entries.flow.async_configure(
             result["flow_id"],
-            {
-                CONF_IP_ADDRESS: IP,
-            },
+            {CONF_IP_ADDRESS: IP, CONF_PASSWORD: ""},
```

Comment:
Why do we send an empty password, is that expected?

Reply:
To be honest, I don't really took the time to understand how voluptuous works. There is an optional field `CONF_PASSWORD` and in real life, a missing password is properly handled for me. In the tests however, there is a `KeyError` if I don't send that explicitly and so I assumed, that voluptuous does that in the background.

Reply:
To be more precise: a device only has a password, if the user configures one. That's why it is optional.